SonarRunner.Shim/RoslynV1SarifFixer.cs

```diff
@@ -93,7 +98,7 @@ private static string ApplyFixToSarif(string unfixedSarif)
         /// 
         /// Returns true if the output is valid SARIF (even if there was no change).
         /// </summary>
-        public static bool FixRoslynV1Sarif(string unfixedSarif, out string output)
+        public /* for test */ bool FixRoslynV1SarifString(string unfixedSarif, out string output)
         {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) In method 'RoslynV1SarifFixer.FixRoslynV1SarifString(string, out string)', correct the spelling of 'Sarif' in parameter name 'unfixedSarif' or remove it entirely if it represents any sort of Hungarian notation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AIdentifiersShouldBeSpelledCorrectly)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Correct the spelling of 'Sarif' in member name 'RoslynV1SarifFixer.FixRoslynV1SarifString(string, out string)' or remove it entirely if it represents any sort of Hungarian notation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AIdentifiersShouldBeSpelledCorrectly)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Consider a design that does not require that 'output' be an out parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AAvoidOutParameters)